daemon/core/nodes/netclient.py

```diff
@@ -126,7 +126,9 @@ def device_flush(self, device):
         :param str device: device to flush
         :return: nothing
         """
-        self.run(f"{IP_BIN} -6 address flush dev {device}")
+        self.run(
+            f"[ -e /sys/class/net/{device} ] && {IP_BIN} -6 address flush dev {device} || true"
+        )
```

Comment:
This would require the shell=True statement, but the shell parameter would need to be plumbed in for node commands, I will take a look at this, when I take a break from the GUI